### ISSUE TITLE
NTR: Small intestine biopsy: adenomas (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9153]
New term request Small intestine biopsy: adenomas (elthat_100527043509): Bowel; intestinal; ampulla of Vater  UMLS: 1302392

**Comment by pnrobinson** [comment]
@MickeySegal
there are many different kinds of adenoma in the small intestine. Could you provide the medical context? Also, is the term specifically about "multiple" adenomas (in which case perhaps we should use the work adenomatosis?)

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/616415
"3 unrelated families in which a total of 7 individuals had multiple colonic adenomas (range, 8-50 adenomas). Four patients developed colorectal cancer. All 3 women had either endometrial hyperplasia or endometrial cancer. Additional more variable findings in these patients included duodenal adenomas,"

https://www.ncbi.nlm.nih.gov/books/NBK1266/
"Adenomas also appear with increased prevalence throughout the GI tract."

**Comment by pnrobinson** [comment]
@MickeySegal 
If we are talking only about PJS, we could title the term
 Peutz-Jaegers-syndrome-type hamartomatous polyps 

?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/616415 is familial adenomatous polyposis-3

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001040
- label: Small intestinal adenoma
- parents: [Neoplasm of the small intestine (HP:0100833)](https://hpo.jax.org/app/browse/term/HP:0100833)
- pmid: PMID:7248908
- def: A small bowel tumor with adenomatous epithelium.
- comment: Grossly and histologically, adenomas arising in the mucosa of the small intestine are similar to the adenomas found in the colon.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9153
- synonyms:
